core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java

```diff
@@ -18,6 +18,15 @@
 
```

Comment:
I don't think the changes in this file are necessary, right?

Can we get this change removed from the PR?